src/main/java/jarvis/commands/UserList.java

```diff
@@ -0,0 +1,146 @@
+package jarvis.commands;
+
+import jarvis.Formatter;
+import jarvis.display.DisplayMessages;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+
+public class UserList extends Task {
+    private static ArrayList<Task> userList= new ArrayList<>(); //list of Tasks
+
+    public UserList(String description) {
+        super(description);
+    }
+
+    public static int getListSize() {
+        return userList.size();
+    }
+
+    public static void insertTask(Task newTask) {
+        userList.add(newTask);
+        DisplayMessages.taskAdded(newTask.getDescription());
+    }
+
+    public static void printList() { //consider converting this function to Jarvis.DisplayMessages
```

Comment:
Avoid writing comments that is not intended for the reader.

Reference: https://nus-cs2113-ay2122s2.github.io/website/schedule/week7/topics.html#implementation-code-quality-comments-basic-write-to-the-reader